Ball of Duty Server/Domain/Entities/Character.cs

```diff
@@ -4,9 +4,32 @@ namespace Ball_of_Duty_Server.Domain.Entities
 {
     public class Character : GameObject
     {
+        public double _score { get; set; }
+        public int HP { get; set; } // PLACEHOLDER FOR THE HP CLASS
+
+        private double SCORE_FACTOR = 0.1;
```

Comment:
this is not a constant, it shouldnt be all upper.
if you want a constant value use the ```const``` keyword. it will then however not be assignable during runtime